xml/deployment_installation.xml

```diff
@@ -715,26 +741,352 @@
          </listitem>
         </varlistentry>
        </variablelist>
        <para>
-        Press <keycap function="enter"
-        />. This boots the system and
-        loads the &productname; installer.
+        Press <keycap function="enter"/>. This boots the system and loads the
+        &productname; installer.
        </para>
       </step>
       <step>
        <para>
         So long as there are no errors, the rest of the installation should
-        complete automatically. After a reboot, the new &worker_node; should
-        show up in the dashboard and can be added to your cluster.
+        complete automatically. After a reboot, the new  node should appear in
+        the dashboard and can be added to your cluster.
        </para>
       </step>
      </procedure>
+     <note>
+      <title>
+       &rootuser; Password
+      </title>
+      <para>
+       When nodes are installed using &ay;, there is no opportunity to specify
+       the password for &rootuser;. However, each node will have <literal>ssh</literal>
+       keys for &rootuser; on the &master_node; pre-installed. Thus it is
+       possible to access the &worker_node;s by <command>ssh</command>ing to
+       them from the &master_node;.
+      </para>
+     </note>
+      
+    </sect4>
+   </sect3>
+   
+   <sect3 xml:id="sec.caasp.installquick.qcow2">
+    <title>Installing from Pre-installed Disk Images</title>
+    <para>
+     For building clusters from virtual machines on supported hypervisors, it
+     is not necessary to individually install each node. &suse; offers
+     pre-installed VM disk images in the following formats: 
+    </para>
+    <variablelist>
+     <varlistentry>
+      <term>&kvm; and &xen;</term>
+      <listitem>
+       <para>
+        In QCOW2 format, for &kvm; and for &xen; using full virtualization.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>&xen;</term>
+      <listitem>
+       <para>
+        In QCOW2 format, for &xen; using paravirtualization.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>VMware</term>
+      <listitem>
+       <para>
+        In VMDK format, for VMware ESXi.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>
+       VHD
+      </term>
+      <listitem>
+       <para>For Microsoft Hyper-V.</para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <sect4>
+     <title>Overview</title>
+     <para>
+      When deploying a cluster node from pre-installed disk images, the setup
+      program never runs. Thus, it is not possible to set values such as time
+      servers, node names, the &rootuser; password for the &admin_node; and so
+      on in the normal way.
+     </para>
+     <para>      
+      Instead, these values must be set using <literal>cloud-init</literal> configuration
+      files, which are placed into a specific subdirectory within an ISO image
```

Comment:
too many whiches: ...ISO image attached to the VM....

Reply:
Good point. Fixed.